.travis.yml

```diff
@@ -1,21 +0,0 @@
-language: go
-
-go:
-  - 1.11.x
-  - 1.12.x
-  - master
-
-services:
-  - docker
-
-script:
-  - 'export GO111MODULE=on'
-  - 'make'
-  - 'make test'
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_1d073d5eb2c7_key -iv $encrypted_1d073d5eb2c7_iv
-    -in scripts/artifactory.lic.enc -out scripts/artifactory.lic -d && make testacc; fi'
```

Comment:
Do you think it would be fair to keep the test portions here so that future PRs still run the test actions? I think its nice to run it right before releasing code as well, but I'd be a little worried about future PRs failing here and then causing bad releases.

Reply:
I've gone and re-added for now. As it appears the actions aren't running. If this is the case for subsequent PRs, I'll keep this in permanently